src/js/msp/MSPHelper.js

```diff
@@ -164,6 +164,16 @@ MspHelper.prototype.process_data = function(dataHandler) {
                     FC.MOTOR_OUTPUT_ORDER[i] = data.readU8();
                 }
                 break;
+            case MSPCodes.MSP2_GET_SMARTAUDIO_DEVICE:
+                FC.SMARTAUDIO_DEVICE = {};
+                FC.SMARTAUDIO_DEVICE.version = data.readU8();
+                FC.SMARTAUDIO_DEVICE.channel = data.readU8();
+                FC.SMARTAUDIO_DEVICE.power = data.readU8();
+                FC.SMARTAUDIO_DEVICE.mode = data.readU8();
+                FC.SMARTAUDIO_DEVICE.freq = data.readU16();
+                FC.SMARTAUDIO_DEVICE.orfreq = data.readU16();
+                FC.SMARTAUDIO_DEVICE.willBootIntoPitMode = Boolean(data.readU8());
```

Comment:
I think we use 0 or 1 rather then Boolean but it's not stored on firmware, so should be ok.
```suggestion
                FC.SMARTAUDIO_DEVICE.willBootIntoPitMode = data.readU8();
```

Reply:
in the firmware it is Boolean i think.
The it's being converted to U8 to transfer via MSP
```
typedef struct smartAudioDevice_s {
    int8_t version;
    int8_t channel;
    int8_t power;
    int8_t mode;
    uint16_t freq;
    uint16_t orfreq;
    bool willBootIntoPitMode;
} smartAudioDevice_t;
```
https://github.com/betaflight/betaflight/blob/master/src/main/io/vtx_smartaudio.h#L71

So i figured it may be better to have boolean on the Configurator part as well :)